src/modules/wreck/job.c

```diff
@@ -758,7 +758,7 @@ static void wreck_state_cb (flux_t *h, flux_msg_handler_t *w,
     const char *kvs_path;
     struct wreck_job *job;
 
-    if (flux_event_unpack (msg, &topic, "{s:I s:s}",
+    if (flux_event_unpack (msg, &topic, "{s:I s?s}",
```

Comment:
initialize kvs_path to NULL in case it's not assigned?

Reply:
Ah, good catch!